examples/add-filter-processor/rendered_manifests/secret-splunk.yaml

```diff
@@ -7,12 +7,12 @@ metadata:
   namespace: default
   labels:
     app.kubernetes.io/name: splunk-otel-collector
-    helm.sh/chart: splunk-otel-collector-0.103.0
+    helm.sh/chart: splunk-otel-collector-0.104.0
     app.kubernetes.io/managed-by: Helm
     app.kubernetes.io/instance: default
-    app.kubernetes.io/version: "0.103.0"
+    app.kubernetes.io/version: "0.104.0"
```

Comment:
Why the upgrade here? Can you please break it down into several PRs? Each PR to serve a particular purpose

Reply:
Sorry about that, accidentally had my git command automation push to the wrong branch. Reverted the chart upgrade changes, only two files are created or updated in this PR.